contrib/models/checklist.py

```diff
@@ -44,13 +46,18 @@ class RefusalReasonCategory(models.Model):
         verbose_name=_('Name'),
     )
 
+    order = models.FloatField(
```

Comment:
Pourquoi des floatfield ?

Reply:
Je me dis que c'est parfois plus pratique lors de l'ajout d'une nouvelle catégorie en particulier si, au départ, toutes les catégories ont une valeur d'ordre entière. Ainsi, par exemple, si on veut ajouter une catégorie entre la catégorie 1 et 2, on met 1.5, ce qui évite de changer les valeurs des catégories existantes. Après on pourrait simplement utiliser les entiers et par défaut espacer l'ordre de chaque catégorie de 10 par exemple pour laisser de la place aux futures catégories.

Reply:
A voir si cela te convient ou si tu préfères que j'utilise plutôt des entiers ?

Reply:
J'avais pas pensé à ca, en effet.
Ca me va du coup

On aurait pu également utiliser django-ordered-model qui permet de déplacer a un index avec .to(<index>)
Je sais pas s'ils compte changer l'ordre eux même a un moment

Reply:
J'ai utilisé `django-ordered-model` finalement.